cromwell_tools/utilities.py

```diff
@@ -300,10 +300,17 @@ def prepare_workflow_manifest(wdl_file, inputs_files=None, options_file=None, de
     # Compose WDL dependencies
     if dependencies:
         if isinstance(dependencies, list):
-            zip_file = make_zip_in_memory(download_to_map(dependencies))
+            if len(dependencies) == 1 and dependencies[0].endswith('.zip'):
```

Comment:
Can you add a unit test for this case?

Reply:
Sure, I think I'm going to add 1 case and modify 1 existing test case. ⚙️